components/ide/code/startup.sh

```diff
@@ -31,4 +31,4 @@ export USER=gitpod
 [ -s ~/.nvm/nvm-lazy.sh ] && source ~/.nvm/nvm-lazy.sh
 
 cd /ide
-exec /ide/node/bin/gitpod-node --inspect=0 ./out/gitpod.js $*
\ No newline at end of file
+exec /ide/node/bin/gitpod-node ./out/gitpod.js $*
```

Comment:
it is a clean up

Reply:
it is confusing, a user sees some port exposed when he does not start anything. I think it is better not to do it.